src/boot/efi/meson.build

```diff
@@ -266,11 +266,19 @@ efi_ldflags = [
         efi_crt0,
 ]
 
-possible_link_flags = [
-        '-Wl,--no-warn-execstack',
-        '-Wl,--no-warn-rwx-segments',
-]
-efi_ldflags += cc.get_supported_link_arguments(possible_link_flags)
+foreach arg : ['-Wl,--no-warn-execstack',
+               '-Wl,--no-warn-rwx-segments']
+        # We need to check the correct linker for supported args. This is what
+        # cc.has_multi_link_arguments() is for, but it helpfully overrides our
```

Comment:
Not a matter, but comment should be `cc.get_supported_link_arguments()`?

Reply:
No, this is correct. `get_supported_link_arguments` tests each arg individually, but we want to test them together with `-fuse-ld=`, which `get_supported_link_arguments` does not support.

Reply:
I see. Thank you!